crates/tools/js/benchmark/solidity-tests.js

```diff
@@ -61,10 +61,18 @@ async function runForgeStdTests(forgeStdRepoPath) {
     .map(loadContract.bind(null, hardhatConfig))
     .filter((ts) => !EXCLUDED_TEST_SUITES.has(ts.id.name));
 
-  const runner = new SolidityTestRunner(gasReport, (...args) => {
-    console.error(`${args[1].name} took ${elapsedSec(start)} seconds`);
+  const results = await new Promise((resolve) => {
```

Comment:
Nice to have: I see some duplication of this code with other call sites. We could provide a helper function in TS that accumulates the results as callbacks are called.

Reply:
Yeah we will definitely want to have a JS wrapper at some point. It's not so clear to handle it now especially assuming it's TS, so I'd defer this. E.g. is the wrapper in `edr_napi` (if yes how does the build tie together) or is it a separate package (how do we release that?).